File: src/main/java/io/github/hotlava03/baclava4j/commands/util/UtilCategory.java
package io.github.hotlava03.baclava4j.commands.util;

import java.util.Arrays;
import java.util.List;

import io.github.hotlava03.baclava4j.commands.Category;
import io.github.hotlava03.baclava4j.commands.CategoryType;
import io.github.hotlava03.baclava4j.commands.Command;

public class UtilCategory implements Category {

    @Override
    public CategoryType getType() {
        return CategoryType.UTIL;
    }

    @Override
    public List<Command> getCommands() {
        return Arrays.asList(
            
        );
    }

}

File: src/main/java/io/github/hotlava03/baclava4j/commands/util/UtilCategory.kt
package io.github.hotlava03.baclava4j.commands.util

import io.github.hotlava03.baclava4j.commands.Category
import io.github.hotlava03.baclava4j.commands.CategoryType
import io.github.hotlava03.baclava4j.commands.Command

class UtilCategory : Category {
    override val type: CategoryType
        get() = CategoryType.UTIL

    override val commands: List<Command>
        get() = listOf()
}